research/src/pages/popup/popover.research.explainer.mdx

```diff
@@ -284,6 +284,7 @@ The above rules mean that a popover, when not "shown", has `display:none` applie
 }
 ```
 
+Be mindful when using other stylesheets that you haven't authored. These may set the `display` value on your popover elements. For example, if you use a `menu` element as a `popover` and you use [normalize.css](https://necolas.github.io/normalize.css/). In this case, normalize.css has a rule that sets `display: flex` on `menu` elements.
```

Comment:
did some more research on this and the latest version of normalize css doesn't have this styling issue.  However, CodePen is using a ridiculously old version (5 vs latest 8.x), so just modifying this text to indicate that without specifically calling them out (well... outside of this clarifying comment). 

```suggestion
Be mindful when using other stylesheets that you haven't authored. These may set the `display` value on your popover elements. For example, if you use a `menu` element as a `popover` and you use earlier versions of [normalize.css](https://necolas.github.io/normalize.css/). In this case, normalize.css has a rule that sets `display: flex` on `menu` elements.
```

Reply:
Ahh good research Scott 👏 
I'd assumed CodePen was using the latest. Thanks for this!

Reply:
as would i, but lol assumptions :)